src/impl_windows.rs

```diff
@@ -16,4 +19,18 @@ impl Mouse {
 
         Mouse::Error
     }
+
+    pub fn set_dpi(monitor: usize) {
+        use winapi::um::shellscalingapi::SetProcessDpiAwareness;
+        use winit::event_loop::EventLoop;
+
+        // forgive me
+        let mut monitors = (*EventLoop::new()).available_monitors();
```

Comment:
This part you can do it with the simple winapi, you don't need to use winit (although I understand how simple it can be), anyway I previously recommended to look at the implementation of winit, here is the way they do it

https://github.com/rust-windowing/winit/blob/master/src/platform_impl/windows/monitor.rs#L108-L119